torch/_inductor/scheduler.py

```diff
@@ -1194,15 +1198,17 @@ def codegen(self):
                 ):
                     self.flush()
                 if device != self.current_device:
-                    if device.type == "cuda":
-                        if self.current_device and self.current_device.type == "cuda":
-                            V.graph.wrapper_code.codegen_cuda_device_guard_exit()
+                    if (
+                        self.current_device
+                        and self.current_device.type in all_triton_backend_name()
```

Comment:
It seems like the main usage of these new abstractions is just varients of "is_any_triton()", could we just add a helper function for this?